atlp-rwanda/e-commerce-ones-and-zeroes-bn
# [Finishes #187355126] The products should stop listing if the expiry

### What does This PR Do
Removing products from listing when Expired

### Description of Task to be completed
if the product expire,The seller recieves the email, with the details of the product which have been expired, name, quantity , date of expiration and the Image of the product.
The product, will be flagged as expired and will no longer be listed on available products of our store.

### How should this be manually tested? 
First, register and verify  as seller, then create the collection, add a product, make sure to set also the expiration date, if the date of expiration reaches, the seller will recieve the email of the expired products.

<img width="1366" alt="Screenshot 2024-05-29 at 23 37 45" src="https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-bn/assets/88989844/30e70959-6a9c-4337-95b6-bad63e61bd40">
